packages/mds-db/schema.ts

```diff
@@ -8,6 +8,8 @@ const AUDITS_TABLE = 'audits'
 const AUDIT_EVENTS_TABLE = 'audit_events'
 const POLICIES_TABLE = 'policies'
 const GEOGRAPHIES_TABLE = 'geographies'
+const POLICY_METADATA_TABLE = 'policy_metadata'
```

Comment:
Merging these `mds-db` changes with the identity column db changes is going to be interesting.